Library/Formula/archivemail.rb

```diff
@@ -0,0 +1,38 @@
+class Archivemail < Formula
+  homepage "http://archivemail.sourceforge.net/"
+  url "https://downloads.sourceforge.net/project/archivemail/archivemail-0.9.0.tar.gz"
+  sha1 "9b04604c97b12062af897c949f65303c54d7f55e"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+
+  def install
+    ENV.prepend_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    system "python", *Language::Python.setup_install_args(libexec)
+    bin.install Dir[libexec/"bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+    man1.install Dir[libexec/"share/man/man1/*"]
+  end
+
+  test do
+    (testpath/"inbox").write <<-EOF.undent
```

Comment:
Use `EOS.undent` and indent the body below to two spaces more than the line before.